diff --git a/node_modules/@flexa/flexa-react-native/android/build.gradle b/node_modules/@flexa/flexa-react-native/android/build.gradle
index f8f9adb..85373d1 100644
--- a/node_modules/@flexa/flexa-react-native/android/build.gradle
+++ b/node_modules/@flexa/flexa-react-native/android/build.gradle
@@ -53,14 +53,14 @@ android {
   packagingOptions {
     pickFirst 'META-INF/versions/9/OSGI-INF/MANIFEST.MF'
   }
-}
 
-sourceSets {
-  main {
-    java.srcDirs += [
-      "generated/java",
-      "generated/jni"
-    ]
+  sourceSets {
+    main {
+      java.srcDirs += [
+        "generated/java",
+        "generated/jni"
+      ]
+    }
   }
 }
 
