diff --git a/node_modules/react-native-reanimated-carousel/src/layouts/parallax.ts b/node_modules/react-native-reanimated-carousel/src/layouts/parallax.ts
index 9b802fe..9886c3c 100644
--- a/node_modules/react-native-reanimated-carousel/src/layouts/parallax.ts
+++ b/node_modules/react-native-reanimated-carousel/src/layouts/parallax.ts
@@ -52,11 +52,13 @@ export function parallaxLayout(
       [-size + parallaxScrollingOffset, 0, size - parallaxScrollingOffset],
     );
 
-    const zIndex = interpolate(
-      value,
-      [-1, 0, 1],
-      [0, size, 0],
-      Extrapolation.CLAMP,
+    const zIndex = Math.round(
+      interpolate(
+        value,
+        [-1, 0, 1],
+        [0, size, 0],
+        Extrapolation.CLAMP,
+      )
     );
 
     const scale = interpolate(
