diff --git a/node_modules/react-native-quick-crypto/android/src/main/java/com/margelo/quickcrypto/QuickCryptoModule.java b/node_modules/react-native-quick-crypto/android/src/main/java/com/margelo/quickcrypto/QuickCryptoModule.java
index b19f79f..98dea42 100644
--- a/node_modules/react-native-quick-crypto/android/src/main/java/com/margelo/quickcrypto/QuickCryptoModule.java
+++ b/node_modules/react-native-quick-crypto/android/src/main/java/com/margelo/quickcrypto/QuickCryptoModule.java
@@ -42,6 +42,7 @@ public class QuickCryptoModule extends ReactContextBaseJavaModule {
       System.loadLibrary("reactnativequickcrypto");
 
       JavaScriptContextHolder jsContext = getReactApplicationContext().getJavaScriptContextHolder();
+      @com.facebook.react.common.annotations.FrameworkAPI
       CallInvokerHolderImpl jsCallInvokerHolder = (CallInvokerHolderImpl) getReactApplicationContext()
                                                   .getCatalystInstance()
                                                   .getJSCallInvokerHolder();
